src/components/SkeletonPage/tests/SkeletonPage.test.tsx

```diff
@@ -43,4 +77,18 @@ describe('<SkeletonPage />', () => {
     const skeletonPage = mountWithAppProvider(<SkeletonPage breadcrumbs />);
     expect(skeletonPage.find(SkeletonBodyText)).toHaveLength(1);
   });
+
+  describe('primaryAction', () => {
+    it('renders SkeletonDisplayText if primaryAction is true', () => {
```

Comment:
Small nitpick, this could be phrased without repeating primaryAction, which is already in the describe statement.

`renders SkeletonDisplayText if true`